website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -4859,6 +5169,10 @@ func init() {
 }
 {{< /highlight >}}
 
+{{< paragraph class="language-rust">}}
+Beam SDK for Rust does not support default coders.
```

Comment:
To me this sounds a bit like the Rust SDK is a missing feature 🤔 maybe the description in 7.2.3 should really go here? To me it's a major selling point of the Rust sdk that you will have compile-time certainty that all your types have proper coders.

Reply:
Section 7.2.3 is about the **default** coders, not about the **standard** coders.
Once a default coders for `T` is set, `PCollection<T>`s later created hold the coder by default.

So, the default coders are kind of runtime feature, which I don't like so much.

Since we have the way (`PCollection::with_coder::<SomeCoder>()`) to overwrite a coder for a single `PCollection<T>`, IMO, we don't need the default coder feature.

I would also like to hear about your idea.